pkg/apis/operator/v1alpha1/validation/garden.go

```diff
@@ -610,33 +615,37 @@ func validateGardenerDashboardConfig(config *operatorv1alpha1.GardenerDashboardC
 	return allErrs
 }
 
-func validateOperation(operation string, garden *operatorv1alpha1.Garden, fldPath *field.Path) field.ErrorList {
+func validateOperation(operations []string, garden *operatorv1alpha1.Garden, fldPath *field.Path) field.ErrorList {
 	var (
 		allErrs       = field.ErrorList{}
+		fldPathOp     = fldPath.Key(v1beta1constants.GardenerOperation)
 		k8sLess134, _ = versionutils.CheckVersionMeetsConstraint(garden.Spec.VirtualCluster.Kubernetes.Version, "< 1.34")
 	)
 
-	if operation == "" {
+	if len(operations) == 0 {
 		return allErrs
 	}
 
-	fldPathOp := fldPath.Key(v1beta1constants.GardenerOperation)
+	operationsSet := sets.New(operations...)
```

Comment:
```suggestion
    // Create a `set` to deduplicate the operations array.
	operations := sets.New(operations...).UnsortedList()
```
Then re-use the `operations` instead of creating a new list with `UnsortedList()` every time.